apps/admin_api/config/test.exs

```diff
@@ -1,12 +1,13 @@
 use Mix.Config
 
+# General application configuration
+config :admin_api, enable_client_auth: true
```

Comment:
How can you define this in an ENV variable if it expects a boolean? Having a check `== "true"` would break @sirn's request about not using logic for ENV variables, no?

Reply:
Should be fine, DeferredConfig can normalize settings.